.gitignore

```diff
@@ -30,5 +30,4 @@ quicktests/overlaying/list_of_quicktests.json
 *.project
 
 # Visual Studio Code
-.settings/
-
+.vscode/
```

Comment:
Let's leave both for a while. Does vsc update automatically? Maybe not everyone wants to have the latest version.